src/mpl/include/mpl_gpu.h

```diff
@@ -95,6 +95,11 @@ static inline int MPL_gpu_query_pointer_is_dev(const void *ptr, MPL_pointer_attr
     return 0;
 }
 
+static inline int MPL_gpu_query_pointer_is_strict_dev(const void *ptr, MPL_pointer_attr_t * attr)
+{
+    return 0;
+}
+
```

Comment:
Is this supposed to be in `mpl_gpu_fallback.c`? The inline function here would conflict with the actual definition, right?

Reply:
Since this is in the `#ifndef MPL_HAVE_GPU` block I think it is effectively the same as being defined in `mpl_gpu_fallback.c`, but I can move it there anyways

Reply:
Done